docs/aerodromes/classc/Nowra.md

```diff
@@ -4,7 +4,111 @@
+
+IFR traffic must depart from the active duty runway.
+
+These spots are physically marked with bidirectional numbers in addition to the standard “H” marking:
+
+* Spot-1 (Taxiway C2)
+* Spot-2 (Taxiway C3)
+* Spot-3 (Taxiway B4)
+* Spot-4 (Taxiway B4)
+* Spot-5 (Taxiway B3)
+* Spot-6 (Taxiway B3)
+* Spot-7 (Taxiway A2)
+* Spot-8 (Taxiway A2)
+
+<figure markdown>
+![NW VFR CRD](img/nwrotaryslots.png){ width="700" }
+<figcaption>NW VFR CRD</figcaption>
+</figure>
+
+
+Helicopter departures for aircraft not requiring the use of a runway shall be conducted from the helicopter spots.  Helicopter spots shall be used by all departures and arrivals. Helicopters must remain at their designated slot until given a clearance from GND or TWR.
+
+Take-offs and landings to the Helicopter Spots should be parallel to the duty RWY direction. Normally the most upwind Helicopter Spot parallel to the RWY is designated for departures, and the downwind Helicopter Spot designated for arrivals. Helicopter Spots have been designed to allow simultaneous arrivals and departures of an adjacent Helicopter Spot.
+
+Post landing at a Helicopter Spot, helicopters shall not vacate the Spot until in receipt of a taxi clearance from ATC (SMC)
+
+### Duty runway Spots
+| Active Runway | Departure Spots (Order of Preference) | Arrival Spots (Order of Preference) |
+|---------------|----------------------------------------|-------------------------------------|
+| Runway 08     | Spot-2 (C3), Spot-1 (C2)               | Spot-8 (A2), Spot-7 (A2)            |
+| Runway 26     | Spot-8 (A2), Spot-7 (A2)               | Spot-2 (C3), Spot-1 (C2)            |
+| Runway 03     | Spot-6 (B3), Spot-5 (B3)               | Spot-4 (B4), Spot-3 (B4)            |
+| Runway 21     | Spot-4 (B4), Spot-3 (B4)               | Spot-6 (B3), Spot-5 (B3)            |
+
+
+## Coordination
+NWA TCU
+
+'Next' coordination is required from NWA ADC to NWA TCU for all aircraft.
+
+Helicopter departing from helicopter spots will be treated as if departing from that duty runway.
+
+!!! example
+    <span class="hotline">**NWA ADC** -> **NWA TCU**</span>: "Next, ASY01, runway 08"  
+    <span class="hotline">**NWA TCU** -> **NWA ADC**</span>: "ASY01, Assigned Heading Left 030, unrestricted"  
+    <span class="hotline">**NWA ADC** -> **NWA TCU**</span>: "Assigned Heading Heading Left 030, ASY01" 
+
+
+### Level Assignment
+The Standard Assignable Level from **NWA ADC** to **NWA TCU** is:
+
+Issue `F130` or `RFL`; whichever is lower, to aircraft cleared via an SID or aircraft with a wake turbulence category of medium or greater.  
+Issue `F130` or `RFL`; whichever is lower, to jet aircraft not cleared via an SID regardless of wake turbulence category.  
+Issue `A030` or `RFL`; whichever is lower, to aircraft not cleared via an SID with a wake turbulence category of light.  
+Issue `A025` or `RFL`; whichever is lower, to VFR traffic.
```

Comment:
We can probably simplify this further. How about F120 (which will be the SAL outbound to ENR) or RFL, whichever is lower, for _all aircraft_? I can't see any operational benefit to splitting the initial climb altitude this way, other than the fact that it's what they seem to do in YPDN.